sympy/sets/sets.py

```diff
@@ -1808,6 +1836,43 @@ def __lt__(self, other):
         return self.is_proper_subset(other)
 
 
+class SymmetricDifference(Set):
+    """Representes the Symmetric Difference of a set with another set.
```

Comment:
See also [this](https://github.com/sympy/sympy/wiki/Writing-documentation) about writing documentation.